app/domain/authentication/authn_jwt/validate_restrictions_one_to_one.rb

```diff
@@ -0,0 +1,19 @@
+module Authentication
+  module AuthnJwt
+
+    # This class is responsible for retrieving the correct value from the GCP token
+    # of the requested attribute.
+    class ValidateRestrictionsOneToOne
+      def initialize(decoded_token:)
+        @decoded_token = decoded_token
+      end
+
+      def valid_restriction?(restriction)
```

Comment:
Can restriction be `null` or something like this?

Reply:
Because we do the restriction extraction from the db there is no case it should be null and this case will lead to error when we trying to access name attribute which will make the authentication request to fail